src/msw/window.cpp

```diff
@@ -4884,6 +4884,38 @@ wxWindowMSW::MSWUpdateOnDPIChange(const wxSize& oldDPI, const wxSize& newDPI)
     // update font if necessary
     MSWUpdateFontOnDPIChange(newDPI);
 
+    // update sizers
+    if ( GetSizer() )
+    {
+        wxSizerItemList::compatibility_iterator current =
+            GetSizer()->GetChildren().GetFirst();
+        while ( current )
```

Comment:
I prefer writing these loops as
```
        for ( wxWindowList::compatibility_iterator node = GetChildren().GetFirst();
              node;
              node = node->GetNext() )
```
but it's pretty minor, of course. A `for` loop would be simpler to replace with ranged for in the future...